src/main/java/com/code_intelligence/jazzer/mutation/engine/SeededPseudoRandom.java

```diff
@@ -267,4 +271,20 @@ private static double zipf_hinv(double x) {
   private static double zipf_pow_negative_q(double x) {
     return 1.0 / (x * x);
   }
+
+  @Override
+  public <T> T pickValue(T value, T otherValue, Supplier<T> producer) {
```

Comment:
While I like the idea of this function, I think that we risk using this in many unrelated places that then all share a single global "use producer" probability. I would feel more comfortable with this if any caller had to consciously pass in a value for the inverse frequency. Otherwise we never know what's going to happen if we ever turn that knob :-)

Reply:
Very valid concern, changed.